website/content/en/docs/deployment/cognito/manifest/guide.md

```diff
@@ -204,8 +204,8 @@ make deploy-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
     Before connecting to the dashboard:
 
     * Go to the Cognito console and create some users in `Users and groups`. These are the users who will log in to the central dashboard.
-        ![cognito-user-pool-created](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/cognito-user-pool-created.png)
-
+        - Create a user with email address user@example.com this user and email address come preconfigured and have a Profile created by default.
```

Comment:
Create a user with email address user@example.com`.` This

Mask ruidi's email in screenshot